docs/source/tutor/_hello_world.ipynb

```diff
@@ -0,0 +1,698 @@
+    "<summary>Not familiar with object-oriented programming glossary: class, object, method and instances?</summary>\n",
+    "\n",
+    "- A **class** is like a blueprint or a template that defines the characteristics and behaviors of a particular type of object. For example, we can define a `Dog` class that includes attributes such as breed, age, and name, as well as behaviors such as barking and wagging its tail.\n",
+    "\n",
+    "- An **object** is an **instance** of a class. So, if we have a `Dog` class, we can create objects of that class, such as \"Fido\" and \"Buddy\". Each object of the `Dog` class will have its own set of **attributes**, such as \"Fido\" being a Golden Retriever and \"Buddy\" being a Chihuahua, and **methods**, such as \"Fido\" barking loudly and \"Buddy\" wagging its tail.\n",
+    "\n",
+    "- An **instance** is a specific occurrence of an **object** created from a **class**. For example, with our `Dog` class, we can create an instance of that class called \"my_dog\" with specific attributes and behaviors.\n",
+    "\n",
+    "- A **method** is a function that is defined in a **class** and can be called on an **object** of that class. For example, the \"bark\" method defined in the `Dog` class can be called on the object \"my_dog\" to make it bark.\n",
+    "</details>"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Follow these steps to load the `bluesky` and `databroker` packages for data collection activities."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "1"
+      ]
+     },
+     "execution_count": 1,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import pathlib, sys, os\n",
+    "my_path = str(pathlib.Path.home() / \"bluesky\")\n",
+    "sys.path.append(my_path)\n",
+    "\n",
+    "import bluesky\n",
+    "import databroker\n",
+    "from bluesky.callbacks.best_effort import BestEffortCallback\n",
+    "\n",
+    "cat = databroker.temp().v2\n",
+    "RE = bluesky.RunEngine()\n",
+    "RE.subscribe(cat.v1.insert)\n",
+    "RE.subscribe(BestEffortCallback())"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<details>\n",
+    "<summary>Line by line explanation</summary>\n",
+    "\n",
+    "- `my_path = ...` define a variable `my_path` as the \"bluesky\" directory located in the user's home directory\n",
+    "- `sys.path.append(my_path)` adds this directory to the list of directories that Python will search when you try to import a module.\n",
+    "- `cat = databroker.temp().v2` creates a temporary databroker instance and assigns it to the variable `cat`. This allows you to interact with the databroker instance and access its functionality, such as querying for data, creating runs, or accessing metadata.\n",
```

Comment:
creates an instance of a databroker _temporary_ catalog (a catalog which is deleted when the python session is exited).